packages/nextjs/next.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  webpack: (config) => {
+    config.module.rules.forEach((rule) => {
+      if (rule.test && rule.test.toString().includes("tsx|ts")) {
+        rule.include = [
+          ...rule.include,
+          // eslint-disable-next-line @typescript-eslint/no-var-requires
+          require("path").join(__dirname, ".."),
+        ];
+      }
+    });
+    return config;
+  },
+};
```

Comment:
Why is this needed? Some comments would help explain (I'm assuming it's related to importing from the components library)